libs/hdf-converters/test/attestations/attestations.spec.ts

```diff
@@ -6,108 +6,371 @@ import {
   advanceDate,
   Attestation,
   createAttestationMessage,
-  parseXLSXAttestations
+  parseXLSXAttestations,
+  convertAttestationToSegment
 } from '../../src/utils/attestations';
 import {omitHDFTimes, omitVersions} from '../utils';
-
-const attestationArray: Attestation[] = [
+import _ from 'lodash';
+// test a new attestation array for each frequency, use arbitrary far date (yr 3000, 1900)
+// try applying to attestations w diff statuses
+const attestationMessageArray: Attestation[] = [
   {
     control_id: 'V-72087',
-    explanation:
-      'Audit logs are automatically backed up and cleared as necessary',
-    frequency: 'monthly',
+    explanation: 'This is a passing attestation that is not expired',
+    frequency: '1d',
     status: ExecJSON.ControlResultStatus.Passed,
-    updated: moment().format('YYYY[-]MM[-]DD'),
-    updated_by: 'Json Smith, Security'
+    updated: '3000-01-01',
+    updated_by: 'John Doe'
   },
   {
     control_id: 'V-73163',
-    explanation: 'Audit records are enabled for this system.',
+    explanation: 'This is a passing attestation that is expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Passed,
+    updated: '1999-01-01',
+    updated_by: 'Jane Doe'
+  },
+  {
+    control_id: 'V-73164',
+    explanation: 'This is a failing attestation that is not expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Failed,
+    updated: '3000-01-01',
+    updated_by: 'John Doe'
+  },
+  {
+    control_id: 'V-73165',
+    explanation: 'This is a failing attestation that is expired',
+    frequency: '1d',
+    status: ExecJSON.ControlResultStatus.Failed,
+    updated: '1999-01-01',
+    updated_by: 'Jane Doe'
+  }
+];
+
+const aPass_cSkip_unexp: Attestation[] = [
+  {
+    control_id: 'SV-230223',
+    explanation:
+      'This is a test unexpired passing attestation for a skipped control',
```

Comment:
```suggestion
      'This control passes according to this attestation',
```

Again, take my suggestion as only one way to do it, but consider what exactly this explanation represents and write it to that.

Reply:
That makes sense. It should at least be written like one would write an actual attestation, as opposed to just putting whatever in there, in order to make the test clear and understandable to a user or developer.